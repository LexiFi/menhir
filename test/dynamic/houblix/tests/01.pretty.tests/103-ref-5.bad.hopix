let main =
  val y = ref "foo";
  ref "bar"