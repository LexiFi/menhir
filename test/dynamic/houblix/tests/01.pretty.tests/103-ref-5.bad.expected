Line 2, characters 8-9: Error (parsing)
  Syntax error.
